pkg/vault/test/foundry/BatchRouterERC4626Pool.t.sol

```diff
@@ -149,38 +146,40 @@ contract BatchRouterERC4626PoolTest is BaseERC4626BufferTest {
             assertEq(
                 afterWaDAIBufferBalanceWrapped,
                 beforeWaDAIBufferBalanceWrapped - exactWrappedAmountsIn[waDaiIdx],
-                "Vault: waDAI wrapped buffer balance should decrease"
+                "Vault: wrong waDAI wrapped buffer balance"
             );
             assertEq(
                 beforeWaDAIBufferBalanceUnderlying,
                 afterWaDAIBufferBalanceUnderlying - exactUnderlyingAmountsIn[waDaiIdx],
-                "Vault: waDAI underlying buffer balance should increase"
+                "Vault: wrong waDAI underlying buffer balance"
             );
         }
         {
-            (, , , uint256[] memory balances) = vault.getPoolTokenInfo(address(erc4626Pool));
-            assertEq(
-                balances[waDaiIdx],
-                erc4626PoolInitialAmount + exactWrappedAmountsIn[waDaiIdx],
-                "ERC4626 Pool: waDAI balance should increase"
+            (, , , uint256[] memory lastBalancesLiveScaled18) = vault.getPoolTokenInfo(address(erc4626Pool));
```

Comment:
hm why are we using live balances here? Shouldn't we be checking raw if we're comparing that against initial amounts / exact amounts in?

Reply:
You're right.. I just renamed it, forgot to check if it made sense. I'll change this.

Reply:
The test is correct. erc4626Pool has wrapped amounts, so the normal balance of the pool is wrapped balance. Since the tokens have 18 decimals, the lastBalancesLiveScaled18 return underlying amounts, which we can compare with the underlying amounts in.